red_lower : 150
red_upper : 256
green_lower : 98
green_upper : 198
blue_lower : 26
blue_upper : 132
